### PR TITLE
Future + Feature (without screenshot)

### PR DESCRIPTION
**Pull request by huongvu2312**
Trying to fix the git flow took me too much time, remove the project and clone it again didn't work, I can't pull any of the old branches. So to stop wasting any more time on this I will just create a new branch to work on, rather than update the current techstack branch.

### CODE REVIEW

File: content/ss23/master/m4-shop-gpt/challenges.md
Comment (kimc0de) [inline content/ss23/master/m4-shop-gpt/challenges.md:1]: this file is already on main like this. why does github considered it updated?

**Comment by kimc0de** [comment]
The title is future and features but it seems like you have only update content for features. Future is already done in main :D

File: content/ss23/master/m4-shop-gpt/challenges.md
Comment (huongvu2312) [reply content/ss23/master/m4-shop-gpt/challenges.md:1]: I did update the weight (order) of each tab.

**Comment by huongvu2312** [comment]
> The title is future and features but it seems like you have only update content for features. Future is already done in main :D

Yeah, I just check my change again. It seems on Friday I updated future change directly in main and updated features on techstack branch? What a mess, I'm so sorry.

**Pull request closed without merging**